exchange/replication.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bufio"
 	"context"
 	"fmt"
+	"github.com/rs/zerolog/log"
```

Comment:
Are those added up there automatically by intellig?

Reply:
Yes but I checked and that was already used somewhere else in the repo so I assumed it was the standard lib we use for logging